finance_runbook.md

```diff
@@ -6,50 +6,22 @@ Generally, this is a core role.
 
 ## Role description
 
-What does this role do? What are you primarily responsible for?
+For Finance, the primary responsibility lies with the core organizers, to establish the budget prior to the conference and keep folks on track to pull off a great conference day-of while leaving enough to get things started again next year (target is $100K remaining after the conference is complete, to make a venue deposit, pay vendors before sponsorships start coming in, and pay for ongoing maintenance and conference development costs).
 
-### General Tasks
-
-Can you name typical tasks? For example:
-
-* Write a compelling CFP (the CFP committee)
-* Requisition a unique annual logo (swag)
-* etc.
-
-## Important Dates
-
-What dates are important for this role? The following headings and details are examples.
-
-### In the beginning
-
-* Come up with good stage jokes.
-
-### 3 Months Out
-
-* Find vendors for your stage costume.
+There is a secondary responsibility which lies with all organizers and volunteers, to keep the entire team updated with a clear understanding of what has been committed to, what has been spent, what reimbursement is needed, and any other information which would affect the financial status of the conference.
 
-### 2 Months Out
-
-* Try on the costume.
-
-### 1 Month Out
-
-* Sell out the sponsorships.
-
-### The Week Before
-
-* Panic.
-
-### The Day Before
+### General Tasks
 
-* Set up.
-* Party.
+* Establish budget (Core)
+* Maintain budget (Everyone)
+* Submit expenses (Them what spent it, via email, add the receipt in the year's Receipts folder)
+* Reimburse expenses (Core, ConferenceOps)
 
```

Comment:
I'd still have the important dates in here:

- Budget set 10 mo out (or pre-kickoff). <-- necessary for sponsor prospectus, venue choice.
- Budget check-ins would be good throughout the season, like maybe before tickets open, 6 mo out, when early bird closes, 3 mo out...
- Books close date (I'd check with Laura W on this one, but I think it's June?)

Reply:
@moutons ^^ thoughts?

Reply:
see the recent updates